docs/bibliography/biblio.bib

```diff
@@ -1,5 +1,5 @@
 %1
```

Comment:
Latex failed to resolve **Sesame** citation during build process, so add it please